qemu/tests/cfg/multi_host.cfg

```diff
@@ -209,6 +209,77 @@
                       kdump_propagate_cmd = "kdumpctl propagate"
                       kdump_restart_cmd = "systemctl restart kdump.service"
                       kdump_status_cmd = "systemctl status kdump.service"
+                - with_xbzrle:
+                    type = migration_multi_host_with_xbzrle
+                    capabilitys = "xbzrle cache_size"
+                    capabilitys_state = "disable disable"
```

Comment:
in framework, get cache_size state of migrate-capability from function "get_migrate_capability", it returns "False". In my code I marked it as disable and convert to False. So this is ok.
And I added function "set_migrate_cache_size" and "get_migrate_cache_size", use them to set and check the cache_size.